portality/tasks/harvester_helpers/workflow.py

```diff
@@ -1,34 +1,56 @@
+from portality.api.v2 import ArticlesCrudApi
 from portality.api.v2.client import client as doajclient
 from portality.core import app
 from portality.lib import plugin
-from portality.decorators import capture_sigterm
+from portality.models import Journal, Account
 from portality.models.harvester import HarvesterProgressReport as Report
 from portality.models.harvester import HarvestState
 from portality.lib.dataobj import DataObjException
+from portality.settings import BASE_FILE_PATH
 
 
 class HarvesterWorkflow(object):
+    logger = None
 
     @classmethod
-    @capture_sigterm
-    def process_account(cls, account_id):
-        app.logger.info("Harvesting for Account:{x}".format(x=account_id))
+    def create_logger(cls):
+        try:
+            cls.logger = open(BASE_FILE_PATH + "/Harvester_logger.log", "x")
+        except:
+            cls.logger = open(BASE_FILE_PATH + "/Harvester_logger.log", "wr")
+            cls.logger.truncate(0)
+        return cls.logger
```

Comment:
For simplicity, I'd consider just keeping the log information in memory, as it's going to get written to the audit logs down the line anyway.  That saves you just writing it to disk.

If you are certain you want to write it to disk, then we should use our file store instead.  There is a Temporary File store implementation which can be used to stage files.

```
from portality import store
import io

tmpStore = store.StoreFactory.tmp()
tmpStore.store("harvester", "Harvester_logger-2021-06-15.log", source_stream=io.StringIO(""))
loggerFile = tmpStore.get("harvester", "Harvester_logger-2021-06-15.log")

loggerFile.write("log line")
...

# then later
tmpStore.delete_file("harvester", "Harvester_logger-2021-06-15.log")